.gitignore

```diff
@@ -9,8 +9,9 @@ dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
 
-# Avoid ignoring Maven wrapper jar file (.jar files are usually ignored)
-!/.mvn/wrapper/maven-wrapper.jar
+# Ignoring Maven wrapper to be downloaded
+.mvn/wrapper/maven-wrapper.jar
```

Comment:
Not sure about this, it seems that **maven-wrapper** is able to download new versions of hitself.
Do we really want to ignore changes to this file ?

Reply:
We do not want to store binaries (which can be downloaded) in the GIT repo.